srcpkgs/freerdp3/template

```diff
@@ -0,0 +1,143 @@
+# Template file for 'freerdp3'
+pkgname=freerdp3
+version=3.5.0
+revision=1
+build_style=cmake
+build_helper=qemu
+configure_args="
+ -DRDTK_FORCE_STATIC_BUILD=ON
+ -DUWAC_FORCE_STATIC_BUILD=ON
+ -DWITH_BINARY_VERSIONING=ON
+
+ -DCHANNEL_URBDRC_CLIENT=ON
+ -DDOCBOOKXSL_DIR=/usr/share/xsl/docbook
+ -DPROXY_PLUGINDIR=/usr/lib/${pkgname}/server/proxy/plugins
+ -DWAYLAND_SCANNER=/usr/bin/wayland-scanner
+ -DXSLTPROC_EXECUTABLE=/usr/bin/xsltproc
+
+ -DPKG_CONFIG_RELOCATABLE=NO
+ -DWITH_ALSA=ON
+ -DWITH_CAIRO=ON
+ -DWITH_CHANNELS=ON
+ -DWITH_CLIENT_CHANNELS=ON
+ -DWITH_CLIENT_SDL=ON
+ -DWITH_CUPS=ON
+ -DWITH_DSP_FFMPEG=ON
+ -DWITH_FFMPEG=ON
+ -DWITH_FREERDP_DEPRECATED_COMMANDLINE=ON
+ -DWITH_FUSE=ON
+ -DWITH_GSTREAMER_1_0=ON
+ -DWITH_ICU=ON
+ -DWITH_JPEG=ON
+ -DWITH_PCSC=ON
+ -DWITH_PULSE=ON
+ -DWITH_SERVER=ON
+ -DWITH_SERVER_CHANNELS=ON
+ -DWITH_SWSCALE=ON
+ -DWITH_WAYLAND=ON
+ -DWITH_WINPR_TOOLS=ON
+ -DWITH_X11=ON
+ -DWITH_XCURSOR=ON
+ -DWITH_XEXT=ON
+ -DWITH_XI=ON
+ -DWITH_XINERAMA=ON
+ -DWITH_XKBFILE=ON
+ -DWITH_XRENDER=ON
+ -DWITH_XV=ON
+
+ -DWITH_LIBSYSTEMD=OFF
+ -DWITH_SAMPLE=OFF"
+hostmakedepends="pkg-config wayland-devel xmlto"
+makedepends="alsa-lib-devel cJSON-devel cairo-devel cups-devel ffmpeg-devel
+ fuse3-devel icu-devel libXcursor-devel libXdamage-devel libXinerama-devel
+ libXrandr-devel libXv-devel libjpeg-turbo-devel libusb-devel libxkbcommon-devel
+ libxkbfile-devel mit-krb5-devel openssl-devel pcsclite-devel
+ pkcs11-helper-devel pulseaudio-devel uriparser-devel wayland-devel"
+short_desc="Free RDP (Remote Desktop Protocol) client"
+maintainer="chrysos349 <chrysostom349@gmail.com>"
+license="Apache-2.0"
+homepage="https://www.freerdp.com/"
+changelog="https://raw.githubusercontent.com/FreeRDP/FreeRDP/3.5.0/ChangeLog"
+distfiles="https://github.com/FreeRDP/FreeRDP/archive/${version}.tar.gz"
+checksum=03323b383980ee91decbed88270bac061ffb17fd04e52576c70da7885601ecbe
+CFLAGS="-Wno-dev"
+
+# help cmake generate manuals when cross-compiling
+post_patch() {
+	vsed -i cmake/InstallFreeRDPMan.cmake \
+		-e '/generate_argument_docbook/s/COMMAND/& \${CMAKE_CROSSCOMPILING_EMULATOR}/'
+}
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		export CXX="$XBPS_WRAPPERDIR/${XBPS_CROSS_TRIPLET}-c++"
+		configure_args+="
+		 -DCMAKE_CXX_COMPILER_AR=/usr/bin/${XBPS_CROSS_TRIPLET}-ar
+		 -DCMAKE_CXX_COMPILER_RANLIB=/usr/bin/${XBPS_CROSS_TRIPLET}-ranlib"
+	fi
+}
```

Comment:
`$CXX` is usually defined to `${XBPS_CROSS_TRIPLET}-c++` automatically, and the wrapper directory should take precedence in the path. Is the first line really required? Also, the `configure_args` appends should be outside of a function.